### ISSUE TITLE
Some titles are missing in rendered result while trying a rST contains only titles

### ISSUE THREAD

**Issue by frantic1048** [retext-project/retext#197]
**OS:** Arch Linux latest
**ReText Version:** 5.3.0 (using PyMarkups 1.0.1)

While I'm putting just titles in rST file, the rendered result will missing some title( whether preview or html/pdf exporting).

Here's file sample: https://gist.github.com/frantic1048/4caaaa7500668d11f3fb
(Sorry for the link, I don't know how to escape line of many ` in md to prevent breaking the code block. For better viewing I put them on gist)

**Appendix:** Just put any text before the first line, the missing titles are rendered correctly.

**Comment by frantic1048** [comment]
Thanks for the FAQ, now I know why the first two lone title not showed in body.

But I found subtitle(the second lone title) is still missing in HTML/PDF output, and title/subtitle both missing in .odt output.

Also I tried LaTeX output(via `rst2latex` command), the subtitle can be found in document.

I dig into the [testing code](https://github.com/retext-project/pymarkups/blob/4fc4f9621e3af54eb09bb94fb9be6a0e3072b204/tests/test_restructuredtext.py#L10-L14) and relative code([restructuredtext.py#L56](https://github.com/retext-project/pymarkups/blob/master/markups/restructuredtext.py#L56), [abstract.py](https://github.com/retext-project/pymarkups/blob/5abf3598e84173b601049ef65f5f8e19470f935f/markups/abstract.py)), subtitle looks like not being considered.

To HTML, it seems does not have a document subtitle semantic. May the subtitle could simply combined with title, and be the part of the document title(something like "title - subtitle") ?

To other document format, I'm not formilar with them. I think the subtitle should at least appear in some similar semantic in the document.
